docs/source/tutorials/GEDI/GEDI_L4A_Example.ipynb

```diff
@@ -0,0 +1,584 @@
+{
```

Comment:
<p>Much appreciated. Thank you Chuck! </p>

Reply:
<p>Sorry, I missed something at the end of the query. It should be like so (but replace the single quotes with backticks):</p><p> </p><pre class="ql-syntax" spellcheck="false">"l2_quality_flag == 1 &amp; l4_quality_flag == 1 &amp; sensitivity &gt; 0.95 &amp; 'geolocation/sensitivity_a2' &gt; 0.95"
</pre>

Reply:
<p>One more thing, as we chatted about in Slack, is to change the last line in the cell to this:</p><p> </p><pre class="ql-syntax" spellcheck="false">job_id or result
</pre><p> </p><p>so that when submitting the job fails, the user will see the error message instead of an empty string for the job_id.</p>

Reply:
Noted. Thanks

Reply:
<p>Do we need a table showing the most likely lat/lon values for the various GEDI products?</p>